src/Microsoft.IdentityModel.Tokens.Saml/SamlTokenUtilities.cs

```diff
@@ -102,5 +104,79 @@ internal static IEnumerable<SecurityKey> GetKeysForTokenSignatureValidation(stri
             }
             return null;
         }
+
+        /// <summary>
+        /// Gets ValueType of the claim from it's Value.
+        /// </summary>
+        /// <param name="value">Represents value of a claim.</param>
+        /// <returns>String representing claim's ValueType.</returns>
+        internal static string GetClaimValueTypeFromValue(object value)
+        {
+            string claimValueType;
+            claimValueType = TokenUtilities.GetClaimValueTypeFromValue(value);
```

Comment:
why not return TokenUtilities.GetClaimValueTypeFromValue, since it seems ok to return null?
not sure what value this method has over just calling TokenUtilities.

Reply:
Good point, I think I wanted to look for any SAML specific ClaimValueTypes like JSONArray etc.. for Jwt. For now we can call TokenUtilities directly. I forgot to clean up.